aries-site/src/pages/index.js

```diff
@@ -1,9 +1,21 @@
 import React, { useEffect } from 'react';
 import { initialize, pageview } from 'react-ga';
+
+import { Box, Image } from 'grommet';
+import { Tile, Tiles } from 'aries-core';
+
 import { Config } from '../../config';
+import { PageLayout } from '../layouts';
+import { data, TileContent, IntroTile } from '../components/home';
 
-import About from './start/about';
-// import Landing from './landing/landing';
+const HomeTiles = ({ ...rest }) => (
+  <Tiles
+    gap="large"
+    columns={{ count: 'fit', size: 'medium' }}
+    rows="medium"
+    {...rest}
+  />
+);
```

Comment:
Recommending we do this for HomeTiles because on small screens, the Grid was getting pushed off of the right side. That is because there wasn't enough room for a column size of "medium". Therefore, on small screens (when there is already only one tile in a row), we should have the tile fill the width of the column.

Additionally, on mobile there seemed to be a large gap in between each tile. In order to make the distance between the tiles and the edge of the window the same as the space between each tile, we can also adjust the gap:

```
const HomeTiles = ({ ...rest }) => {
  const size = React.useContext(ResponsiveContext);

  return (
    <Tiles
      gap={size !== 'small' ? 'large' : 'medium'}
      columns={{ count: 'fit', size: size !== 'small' ? 'medium' : 'fill' }}
      rows="medium"
      {...rest}
    />
  );
};
```

Reply:
Make sure to import ResponsiveContext in the imports since we don't have it there yet.

Reply:
done